MdeModulePkg/Application/CxlFirmwareMgmt/CxlFirmwareMgmt.inf

```diff
@@ -0,0 +1,42 @@
+## @file
+#
+#  CxlDxe Application setup file
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = CxlFirmwareMgmt
+  FILE_GUID                      = 05092440-531A-4D13-B3F2-B91EEDA9B95F
+  MODULE_TYPE                    = UEFI_APPLICATION
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = MyShellCEntryLib
+
+[Sources]
+  CxlFirmwareMgmt.c
+
+[Packages]
+ ShellPkg/ShellPkg.dec
```

Comment:
MdeModulePkg module can't depend on ShellPkg.

Reply:
@lgao4, Thanks for Review, I am working on it.

Reply:
Review comment is taken care in updated patch. ShellPkg is removed.